### PR TITLE
Update issue management actions

### PR DESCRIPTION
**Pull request by ov7a**
- use custom action for feedback and issues/prs labels
- close stale PRs from the team
- requeue stale contributor PRs

Testing:
https://github.com/ov7a/issue-management-test

### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

File: .github/workflows/stale-pr.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          exempt-all-milestones: true
+
+          # ISSUES (deactivated) ----------------------------------------------
+          # This workflow should touch no issues, so times are set to -1
+          # (see actions/stale documentation for the behavior)
+          days-before-issue-stale: -1
+          stale-issue-label: stale
+          stale-issue-message: >
+            **BUG!** This issue should not be marked stale by the "stale" workflow.
+            Please report it to @bt-developer-productivity
```
Comment (cobexer) [inline .github/workflows/stale-pr.yml:27]: ❓ Should reference `@bt-support` instead

File: .github/workflows/stale-pr.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          days-before-issue-close: -1
+          close-issue-message: >
+            **BUG!** This issue should not be closed by the "stale" workflow.
+            Please report it to @bt-developer-productivity
```
Comment (cobexer) [inline .github/workflows/stale-pr.yml:31]: ❓ Should reference `@bt-support` instead

File: .github/workflows/feedback.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+jobs:
+  feedback:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: gradle/issue-management-action@v1
+        with:
+          script: "feedback"
```
[the remark this answers is not in the document]
Comment (ov7a) [reply .github/workflows/feedback.yml:16]: You are correct.

The action covers p. 1,4, and 5 from
https://github.com/gradle/issue-management-action/issues/3#issuecomment-1886426360
p. 2 and 3 are covered by stale action

**Review by cobexer — CHANGES_REQUESTED** [review]

File: .github/workflows/stale-pr.yml (hunk 1 above)
Comment (ov7a) [reply .github/workflows/stale-pr.yml:27]: Right, it was taken from the old template

File: .github/workflows/issue-metadata.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+jobs:
+  check_issue_metadata:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: gradle/issue-management-action@v1
+        with:
+          script: "issue-metadata"
```
[the remark this answers is not in the document]
Comment (ov7a) [reply .github/workflows/issue-metadata.yml:15]: I have doubts. 
- The entry point is index.ts. The sole purpose of it is to launch one of three scripts: feedback loop, issue metadata check, or PR metadata check.
- The intent of each script is captured in its filename. Each has the same entry point (`run`)
- I don't think we need to capture the imperative action in the method name as we care more about doing the checks, and our action if something off (adding to-triage label) might change

File: .github/workflows/issue-metadata.yml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+name: Check issues metadata
+
+on:
+  issues:
+    types: [ opened, unlabeled, closed ]
```
[the remark this answers is not in the document]
Comment (ov7a) [reply .github/workflows/issue-metadata.yml:5]: Do you have some use case in mind when it would be helpful? I don't see one.

**Review by cobexer — APPROVED** [review]

File: .github/workflows/feedback.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (ov7a) [reply .github/workflows/feedback.yml:18]: Added the link

**Comment by ov7a** [comment]
@bot-gradle merge

**Pull request merged**